Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -12074,8 +12074,6 @@ void WebPageProxy::queryPermission(const ClientOrigin& clientOrigin, const Permi
     } else if (descriptor.name == PermissionName::Geolocation) {
 #if ENABLE(GEOLOCATION)
         name = "geolocation"_s;
```

Comment:
Maybe document that `shouldChangeDeniedToPrompt` is done in WebProcess directly?